languages/ruby/concepts/nil/about.md

```diff
@@ -1 +1,63 @@
-TODO: add information on nil concept
+## Key Points:
+
+- When a class' `.new` method is called to create an object instance, the `.initialize` method is passed all arguments to initialize the instance's state.
+- instance variable names are prefixed with `@`.
+- instance variables default to `nil` until they are explicitly set.
+- instance variables are private by default, and they should be manipulated with getters and setters
+
+```ruby
+class Backpack
+  initialize(owner)
+    @owner = owner
+  end
+
+  def owner
+    @owner
+  end
+
+  def owner=(new_owner)
+    @owner = new_owner
+  end
+end
+```
+
+- Methods named with a trailing `=` are recognized as setters by ruby, and allow the optional use of the assignment syntax, e.g. `Backpack.new("Sven").owner = "Ayah"`
+- Getters and setters can be shortened using the `attr_reader`, `attr_writer`, and `attr_accessor` methods:
```

Comment:
Not sure how to express this, either, but getters and setters aren't shortened by those methods, but they are programmatically written by those methods.